Sources/ResourceLocation/ResourceLocation.swift

```diff
@@ -48,6 +48,9 @@ public enum ResourceLocation: Hashable, CustomStringConvertible, Codable {
     }
     
     public static func from(_ string: String, headers: [String: String] = [:]) throws -> ResourceLocation {
+        if let decoded = try? JSONDecoder().decode(self, from: Data(string.utf8)) {
+            return decoded
+        }
```

Comment:
The purpose of this method was to map a `string` to `ResourceLocation`. This change indeed follows this aim. 

But I've noticed that `headers` argument here. It really should be a part of `string`. So please remove it.

Also please add an unit test for your change into `ResourceLocationTests`. E.g.

```swift
    func test__decoding_string_with_headers() throws {
        let value = """
        {"url": "https://example.url", "headers": {"h1": "v1"}}
        """
        let decoded = try ResourceLocation.from(value)
        XCTAssertEqual(
            decoded,
            ResourceLocation.remoteUrl(URL(string: "https://example.url")!, ["h1": "v1"])
        )
    }
```

Reply:
Fixed this